doc/manual/src/installation/installing-binary.md

```diff
@@ -31,8 +31,8 @@ $ sh <(curl -L https://nixos.org/nix/install) --no-daemon
 ```
 
 This will perform a single-user installation of Nix, meaning that `/nix`
-is owned by the invoking user. You should run this under your usual user
-account, *not* as root. The script will invoke `sudo` to create `/nix`
+is owned by the invoking user. You can run this under your usual user
+account or root. The script will invoke `sudo` to create `/nix`
```

Comment:
"or root" seems to only be the case on Linux (as seen in the conditional on line 903 in this diff), so should that be explicitly called out here?

Reply:
Good point. I'd love to try this on an actual mac and see what happens...

Reply:
I just tried this on a Mac VM (x86), and it worked just fine running as root using `env EUID=65534`, so the check can probably be dropped altogether?

![image](https://user-images.githubusercontent.com/28582702/183716078-79c6ea85-40de-4f0e-8403-7a6f59797ea4.png)

Reply:
Fixed up this review feedback by allowing installing as root on macOS too.